jenkins/ci.suse.de/cloud-ardana8.yaml

```diff
@@ -131,20 +131,42 @@
         - '{ardana_job}'
 
 - project:
-    name: cloud-ardana8-job-std-3cp-devel-staging-updates-x86_64
+    name: cloud-ardana8-job-entry-scale-kvm-x86_64
+    ardana_job: '{name}'
+    ardana_env: cloud-ardana-ci-slot
+    cloudsource: stagingcloud8
+    scenario_name: entry-scale-kvm
+    clm_model: standalone
+    controllers: '3'
+    sles_computes: '2'
+    rhel_computes: '0'
+    ses_enabled: true
+    ses_rgw_enabled: false
+    tempest_filter_list: "\
```

Comment:
Why use ugly backslash based continuation here when you could use `>-`?

Reply:
You're right, it hasn't occurred to me that we can use `>-`. Done